backend/utils.js

```diff
@@ -1,39 +0,0 @@
-/*global process*/
-
-const Bcrypt = require('bcrypt');
-const nodemailer = require('nodemailer');
-const SALT_WORK_FACTOR = 10;
-const hash = (pass, cb) => {
-
-  Bcrypt.genSalt(SALT_WORK_FACTOR, function(error, salt) {
-    if (error) {
-      // eslint-disable-next-line no-console
-      console.log('genSalt :',error)
-
-      throw error
-    }
-    Bcrypt.hash(pass, salt, cb);
-  });
-}
-
-const sendemail = (sender, recipient, sub, content,cb) => {
```

Comment:
The email will only be sent to you after the Admin's approval. :) 
we have approved your request so you should be receiving the confirmation email now!!